<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importar</class>
 <widget class="QWidget" name="importar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>importarDatos</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Importar Datos</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="examinarBtn">
     <property name="text">
      <string>Examinar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="rutaText"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="importarBtn">
     <property name="text">
      <string>importar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
